### PR TITLE
Add mention about disabling getrandom for wasm32-unknown-unknown

### PR DESCRIPTION
**Pull request by golddranks**
Add mention about disabling getrandom for wasm32-unknown-unknown in spirit of https://github.com/rust-random/rand/pull/1250

### CODE REVIEW

File: src/crates.md [hunk 1]
```diff
@@ -46,10 +46,13 @@ which, together with the seed, can be used to reconstruct the generator's state.
...
-require enabling features for WASM support](https://github.com/rust-random/getrandom#features).
+require enabling features for WASM support](https://github.com/rust-random/getrandom#features),
+mainly for seeding entropy via the OS-provided APIs.
 Consequently, if you are using another crate depending on `rand_core` (such as
-most Rand crates), you may have to explicitly enable `getrandom` features for it
-to work.
+most Rand crates), you may have to explicitly enable `getrandom` features for seeding
+to work. Alternatively, in case you are developing for a sandboxed or unknown
+WASM platform (`wasm32-unknown-unknown`) that can't depend on environment provided
+APIs, you might want to disable `getrandom` and seed the generator manually.
```
Comment (dhardy) [inline src/crates.md:55]: I don't like the wording here much: it's the `getrandom` crate which doesn't support `wasm32-unknown-unknown`, and `rand_core` (thus also `rand`) are dependants of that.

Additionally, I'm not sure everyone will follow how to disable `getrandom`?

> you may have to explicitly enable `getrandom` features

Also somewhat unclear — just link to the getrandom doc.

Comment (golddranks) [reply src/crates.md:55]: Ahh, actually I thought it's a bit janky too, but I wasn't too sure about changing the wording too much as I'm not too well-versed in `rand`. I'll rephrase it a bit to my best knowledge!

Comment (golddranks) [reply src/crates.md]: Rephrased it!

**Review by dhardy — APPROVED** [review]
Excellent!

**Pull request merged**